lib/shared/cliOptions.js

```diff
@@ -47,12 +47,6 @@ module.exports = {
     desc: chalk.gray(
       'Print the task dependency tree for the loaded gulpfile.'),
   },
-  depth: {
```

Comment:
This is a breaking change. Can we leave it in as an alias? Does yargs support aliases in the definition?